artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/queue/UpdateQueue.java

```diff
@@ -31,7 +31,7 @@
    String name;
 
    @Option(name = "--deleteOnNoConsumers", description = "whether to delete when it's last consumers disconnects)")
-   Boolean deleteOnNoConsumers = null;
+   String deleteOnNoConsumers = null;
```

Comment:
It is not a personal taste.  It is about consistence along the CLI which is our front end. 


If you want to change this.  Change it everywhere not just here.  And you would need to document it.